testApp/testApp.pro

```diff
@@ -10,7 +10,9 @@ TEMPLATE = app
 
 INCLUDEPATH += $$PWD/../telegram-qt
 
-LIBS += -lssl -lcrypto -lz
+!win32 {
```

Comment:
In Visual Studio I managed this dependencies via nuget and it automatically donwloads the proper version.

Anyway this should not be necessary. It's only to help me to build it on Visual Studio letting nuget to download, include and link dependecies.